### PR TITLE
Updates example code. Minor updates to README and news

### PR DESCRIPTION
**Pull request by adamburkegh**
… in news. Fix minor typo in README

### CODE REVIEW

**Comment by jamesmudd** [comment]
This looks great to me. Thanks for taking the time to make a pull request. Unfortunately I can't merge this yet, hopefully @jimbaker can.

File: index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ import org.python.util.PythonInterpreter;
 public class JythonHelloWorld {
   public static void main(String[] args) {
     try(PythonInterpreter pyInterp = new PythonInterpreter()) {
-      pyInterp.exec("print 'Hello Python World!'");
+      pyInterp.exec("print ('Hello Python World!')");
```
Comment (jamesmudd) [inline index.md]: Should we remove the space between`print` and the `(`

Comment (adamburkegh) [reply index.md]: Done

**Comment by jamesmudd** [comment]
I would like this to be merged, thanks for the work. Unfortunately I can't yet hopefully soon...

**Pull request merged**